internal/pkg/service/stream/mapping/recordctx/fasthttp.go

```diff
@@ -83,7 +83,7 @@ func (c *fastHTTPContext) HeadersMap() *orderedmap.OrderedMap {
 }
 
 func (c *fastHTTPContext) ReleaseBuffers() {
-	c.req.ResetBody()
+	c.req.Request.ResetBody()
```

Comment:
Turns out that `c.req.ResetBody()` actually calls `ctx.Response.ResetBody()`. We want to reset the request body here.

Reply:
https://github.com/keboola/keboola-as-code/pull/2058/files#r1778147284 I am not sure. Internally this calls the `c.req.Request.ResetBody()` but only when `keepBodyBuffer` is on. Are we sure that this eradicate the issue? I think with this approach we will perform another benchmark and conclude that it does not need to help

Reply:
> Internally this calls the `c.req.Request.ResetBody()`

It doesn't. You're in Response object here.

> Are we sure that this eradicate the issue? 

No. I don't think the issue is related to #2058 in the first place.